Prace_domowe/Praca_domowa2/Gryszkiewicz_Maciej/PD2_Gryszkiewicz_Maciej.ipynb

```diff
@@ -0,0 +1,1965 @@
+{
```

Comment:
<p>To prawda :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>